net/portmapper/upnp.go

```diff
@@ -311,6 +311,8 @@ func (c *Client) getUPnPPortMapping(
 
 type uPnPDiscoResponse struct {
 	Location string
+	Server   string
```

Comment:
add example values in comments on what what these are. I have no clue what a USN is. And Server sounds like it's a URL to the server.